packages/iov-bns/scripts/tohex

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+# take input from a file if given, otherwise stdin
+IN=${1:--}
+xxd -p -c 999999 < "$IN"
```

Comment:
`xxd -p -c 999999 < "-"` will not do what you want. Do we need both features?

Reply:
Fair enough, will simplify